content/events/2019/11/2019-11-19-jez-humble-building-scaling-high-performing-technology-organizations.md

```diff
@@ -1,24 +1,27 @@
 ---
 slug: jez-humble-building-scaling-high-performing-technology-organizations
-title: 'Jez Humble&#58; Building and Scaling High Performing Technology Organizations'
-summary: 'This talk presents the findings of that research, including how to measure the performance of software teams, what capabilities organizations should invest in to drive higher performance, and how software leaders can apply these findings in their own organizations&#46;
-'
-featured_image: 
-  uid: 
+title: "Jez Humble&#58; Building and Scaling High Performing Technology Organizations"
+summary: "This talk presents the findings of that research, including how to measure the performance of software teams, what capabilities organizations should invest in to drive higher performance, and how software leaders can apply these findings in their own organizations&#46;"
+featured_image:
+  uid: devops-jez-humble
   alt: ''
-event_type: 
+event_type:
   - zoom
 date: 2019-11-19 13:00:00 -0500
 end_date: 2019-11-19 14:00:00 -0500
 event_organizer: DigitalGov University
-host: DevOps COP 
+host: DevOps Community
 registration_url: https://www.eventbrite.com/e/jez-humble-building-and-scaling-high-performing-technology-organizations-registration-80579640875
 
 ---
 
+{{< img src="devops-jez-humble" >}}
+
 How can we apply technology to drive business value? For years, we've been told that the performance of software delivery teams doesn't matter―that it can't provide a competitive advantage to our companies.
 
-Through four years of groundbreaking research to include data collected from the State of DevOps reports, **Jez Humble** and his colleagues set out to find a way to measure software delivery performance―and what drives it―using rigorous statistical methods. 
+{{< img-right src="jez-humble" >}}
+
+Through four years of groundbreaking research to include data collected from the State of DevOps reports, **Jez Humble** and his colleagues set out to find a way to measure software delivery performance―and what drives it―using rigorous statistical methods.
 
 This talk presents the findings of that research, including how to measure the performance of software teams, what capabilities organizations should invest in to drive higher performance, and how software leaders can apply these findings in their own organizations.
 
```

Comment:
```
**About the speaker**: Jez Humble is co-author of the Shingo Award-winning Accelerate, The DevOps Handbook, Lean Enterprise, and the Jolt Award-winning Continuous Delivery. He has spent his career tinkering with code, infrastructure, and product development in companies of varying sizes across three continents, including [work for the US Federal Government](https://18f.gsa.gov/2016/08/10/patterns-for-managing-multi-tenant-cloud-environments/) at [18F](https://18f.gsa.gov). As well as serving as DevOps Research and Assessment’s (DORA) CTO, Jez teaches at UC Berkeley.
```

Reply:
Where it says, _"including work for the US Federal Government"_, can we mention 18F in there as well?

Reply:
Oh wait, it does.

Reply:
Bio updated and added in 👍